other-examples/collector/confluentcloud/collector.yaml

```diff
@@ -0,0 +1,55 @@
+receivers:
+  kafkametrics:
+    brokers:
+      - $CLUSTER_BOOTSTRAP_SERVER
+    protocol_version: 2.0.0
+    scrapers:
+      - brokers
+      - topics
+      - consumers
+    auth:
+      tls:
+        ca_file: ca.pem
+        cert_file: cert.pem
+        key_file: key.pem
+    collection_interval: 30s
+
+
+  prometheus:
+    config:
+      scrape_configs:
+        - job_name: "confluent"
+          scrape_interval: 60s # Do not go any lower than this or you'll hit rate limits
+          static_configs:
+            - targets: ["api.telemetry.confluent.cloud"]
+          scheme: https
+          basic_auth:
+            username: $CONFLUENT_API_ID
+            password: $CONFLUENT_API_SECRET
```

Comment:
Hey can you explore other options for authenticating besides basic auth for the prometheus receiver? Seems unlikely that this is the recommended best practice according to Confluent. If it is, then fine, but we should do some research to do see if we can do better since basic authentication is generally not advisable.

Reply:
We discussed and think that basic auth is fine here because we're using `scheme: https` and the parameters should be encrypted.